core/src/main/java/org/infinispan/dataconversion/ProtostreamJsonTranscoder.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.query.remote.impl.dataconversion;
+package org.infinispan.dataconversion;
```

Comment:
Why do the transcoders need to move to core? Nothing against it, but It seems they are not used in the persistence layer.